### ISSUE TITLE
Could not find the doublet

### ISSUE THREAD

**Issue by lpatiny** [cheminfo/multiplet-analysis#31]
```
{
  "x": [
    4.004503921169385,
    4.004621040617598,
    4.004738160065812,
    4.004855279514025,
    4.004972398962239,
    4.005089518410452,
    4.005206637858666,
    4.005323757306879,
    4.005440876755093,
    4.005557996203306,
    4.00567511565152,
    4.005792235099733,
    4.0059093545479465,
    4.00602647399616,
    4.006143593444373,
    4.006260712892587,
    4.0063778323408,
    4.006494951789014,
    4.006612071237227,
    4.006729190685441,
    4.006846310133654,
    4.006963429581868,
    4.007080549030081,
    4.007197668478295,
    4.007314787926508,
    4.0074319073747215,
    4.007549026822935,
    4.0076661462711485,
    4.007783265719362,
    4.007900385167575,
    4.008017504615789,
    4.008134624064002,
    4.008251743512216,
    4.008368862960429,
    4.008485982408643,
    4.008603101856856,
    4.00872022130507,
    4.008837340753283,
    4.008954460201497,
    4.00907157964971,
    4.0091886990979235,
    4.009305818546137,
    4.0094229379943505,
    4.009540057442564,
    4.009657176890777,
    4.009774296338991,
    4.009891415787204,
    4.010008535235418,
    4.010125654683631,
    4.010242774131845,
    4.010359893580058,
    4.010477013028272,
    4.010594132476485,
    4.010711251924699,
    4.010828371372912,
    4.0109454908211255,
    4.011062610269339,
    4.011179729717552,
    4.011296849165766,
    4.011413968613979,
    4.011531088062193,
    4.011648207510406,
    4.01176532695862,
    4.011882446406833,
    4.011999565855047,
    4.01211668530326,
    4.012233804751474,
    4.012350924199687,
    4.0124680436479006,
    4.012585163096114,
    4.0127022825443275,
    4.012819401992541,
    4.012936521440754,
    4.013053640888968,
    4.013170760337181,
    4.013287879785395,
    4.013404999233608,
    4.013522118681822,
    4.013639238130035,
    4.013756357578249,
    4.013873477026462,
    4.013990596474676,
    4.014107715922889,
    4.0142248353711025,
    4.014341954819316,
    4.0144590742675295,
    4.014576193715743,
    4.014693313163956,
    4.01481043261217,
    4.014927552060383,
    4.015044671508597,
    4.01516179095681,
    4.015278910405024,
    4.015396029853237,
    4.015513149301451,
    4.015630268749664,
    4.015747388197878,
    4.015864507646091,
    4.0159816270943045,
    4.016098746542518,
    4.016215865990731,
    4.016332985438945,
    4.016450104887158,
    4.016567224335372,
    4.016684343783585,
    4.016801463231799,
    4.016918582680012,
    4.017035702128226,
    4.017152821576439,
    4.017269941024653,
    4.017387060472866,
    4.0175041799210796,
    4.017621299369293,
    4.0177384188175065,
    4.01785553826572,
    4.017972657713933,
    4.018089777162147,
    4.01820689661036,
    4.018324016058574,
    4.018441135506787,
    4.018558254955001,
    4.018675374403214,
    4.018792493851428,
    4.018909613299641,
    4.019026732747855,
    4.019143852196068,
    4.0192609716442815,
    4.019378091092495,
    4.0194952105407085,
    4.019612329988922,
    4.019729449437135,
    4.019846568885349,
    4.019963688333562,
    4.020080807781776,
    4.020197927229989,
    4.020315046678203,
    4.020432166126416,
    4.02054928557463,
    4.020666405022843,
    4.020783524471057,
    4.02090064391927,
    4.0210177633674835,
    4.021134882815697,
    4.02125200226391,
    4.021369121712124,
    4.021486241160337,
    4.021603360608551,
    4.021720480056764,
    4.021837599504978,
    4.021954718953191,
    4.022071838401405,
    4.022188957849618,
    4.022306077297832,
    4.022423196746045,
    4.022540316194259,
    4.022657435642472,
    4.0227745550906855,
    4.022891674538899,
    4.023008793987112,
    4.023125913435326,
    4.023243032883539,
    4.023360152331753,
    4.023477271779966,
    4.02359439122818,
    4.023711510676393,
    4.023828630124607,
    4.02394574957282,
    4.024062869021034,
    4.024179988469247,
    4.0242971079174605,
    4.024414227365674,
    4.0245313468138875,
    4.024648466262101,
    4.024765585710314,
    4.024882705158528,
    4.024999824606741,
    4.025116944054955,
    4.025234063503168,
    4.025351182951382,
    4.025468302399595,
    4.025585421847809,
    4.025702541296022,
    4.025819660744236,
    4.025936780192449,
    4.0260538996406625,
    4.026171019088876,
    4.0262881385370894,
    4.026405257985303,
    4.026522377433516,
    4.02663949688173,
    4.026756616329943,
    4.026873735778157,
    4.02699085522637,
    4.027107974674584,
    4.027225094122797,
    4.027342213571011,
    4.027459333019224,
    4.027576452467438,
    4.027693571915651,
    4.0278106913638645,
    4.027927810812078,
    4.028044930260291,
    4.028162049708505,
    4.028279169156718,
    4.028396288604932,
    4.028513408053145,
    4.028630527501359,
    4.028747646949572,
    4.028864766397786,
    4.028981885845999,
    4.029099005294213,
    4.029216124742426,
    4.0293332441906395,
    4.029450363638853,
    4.0295674830870665,
    4.02968460253528,
    4.029801721983493,
    4.029918841431707,
    4.03003596087992,
    4.030153080328134,
    4.030270199776347,
    4.030387319224561,
    4.030504438672774,
    4.030621558120988,
    4.030738677569201,
    4.030855797017415,
    4.030972916465628,
    4.0310900359138415,
    4.031207155362055,
    4.0313242748102684,
    4.031441394258482,
    4.031558513706695,
    4.031675633154909,
    4.031792752603122,
    4.031909872051336,
    4.032026991499549,
    4.032144110947763,
    4.032261230395976,
    4.03237834984419,
    4.032495469292403,
    4.032612588740617,
    4.03272970818883,
    4.0328468276370435,
    4.032963947085257,
    4.03308106653347,
    4.033198185981684,
    4.033315305429897,
    4.033432424878111,
    4.033549544326324,
    4.033666663774538,
    4.033783783222751,
    4.033900902670965,
    4.034018022119178,
    4.034135141567392,
    4.034252261015605,
    4.0343693804638185,
    4.034486499912032,
    4.0346036193602455,
    4.034720738808459,
    4.034837858256672,
    4.034954977704886,
    4.035072097153099,
    4.035189216601313,
    4.035306336049526,
    4.03542345549774,
    4.035540574945953,
    4.035657694394167,
    4.03577481384238,
    4.035891933290594,
    4.036009052738807,
    4.0361261721870205,
    4.036243291635234,
    4.0363604110834475,
    4.036477530531661,
    4.036594649979874,
    4.036711769428088,
    4.036828888876301,
    4.036946008324515,
    4.037063127772728,
    4.037180247220942,
    4.037297366669155,
    4.037414486117369,
    4.037531605565582,
    4.037648725013796,
    4.037765844462009,
    4.0378829639102225,
    4.038000083358436,
    4.038117202806649,
    4.038234322254863,
    4.038351441703076,
    4.03846856115129,
    4.038585680599503,
    4.038702800047717,
    4.03881991949593,
    4.038937038944144,
    4.039054158392357,
    4.039171277840571,
    4.039288397288784,
    4.0394055167369975,
    4.039522636185211,
    4.0396397556334245,
    4.039756875081638,
    4.039873994529851,
    4.039991113978065,
    4.040108233426278,
    4.040225352874492,
    4.040342472322705,
    4.040459591770919,
    4.040576711219132,
    4.040693830667346,
    4.040810950115559,
    4.040928069563773,
    4.041045189011986,
    4.0411623084601995,
    4.041279427908413,
    4.0413965473566265,
    4.04151366680484,
    4.041630786253053,
    4.041747905701267,
    4.04186502514948,
    4.041982144597694,
    4.042099264045907,
    4.042216383494121,
    4.042333502942334,
    4.042450622390548,
    4.042567741838761,
    4.042684861286975,
    4.042801980735188,
    4.0429191001834015,
    4.043036219631615,
    4.043153339079828,
    4.043270458528042,
    4.043387577976255,
    4.043504697424469,
    4.043621816872682,
    4.043738936320896,
    4.043856055769109,
    4.043973175217323,
    4.044090294665536,
    4.04420741411375,
    4.044324533561963,
    4.0444416530101766,
    4.04455877245839,
    4.0446758919066035,
    4.044793011354817,
    4.04491013080303,
    4.045027250251244,
    4.045144369699457,
    4.045261489147671,
    4.045378608595884,
    4.045495728044098,
    4.045612847492311,
    4.045729966940525,
    4.045847086388738,
    4.045964205836952,
    4.046081325285165,
    4.0461984447333785,
    4.046315564181592,
    4.0464326836298055,
    4.046549803078019,
    4.046666922526232,
    4.046784041974446,
    4.046901161422659,
    4.047018280870873,
    4.047135400319086,
    4.0472525197673,
    4.047369639215513,
    4.047486758663727,
    4.04760387811194,
    4.047720997560154,
    4.047838117008367,
    4.0479552364565805,
    4.048072355904794,
    4.048189475353007,
    4.048306594801221,
    4.048423714249434,
    4.048540833697648,
    4.048657953145861,
    4.048775072594075,
    4.048892192042288,
    4.049009311490502,
    4.049126430938715,
    4.049243550386929,
    4.049360669835142,
    4.0494777892833556,
    4.049594908731569,
    4.0497120281797825,
    4.049829147627996,
    4.049946267076209,
    4.050063386524423,
    4.050180505972636,
    4.05029762542085,
    4.050414744869063,
    4.050531864317277,
    4.05064898376549,
    4.050766103213704,
    4.050883222661917,
    4.051000342110131,
    4.051117461558344,
    4.0512345810065575,
    4.051351700454771,
    4.0514688199029845,
    4.051585939351198,
    4.051703058799411,
    4.051820178247625,
    4.051937297695838,
    4.052054417144052,
    4.052171536592265,
    4.052288656040479,
    4.052405775488692,
    4.052522894936906,
    4.052640014385119,
    4.052757133833333,
    4.052874253281546,
    4.0529913727297595,
    4.053108492177973,
    4.053225611626186,
    4.0533427310744,
    4.053459850522613,
    4.053576969970827,
    4.05369408941904,
    4.053811208867254,
    4.053928328315467,
    4.054045447763681,
    4.054162567211894,
    4.054279686660108,
    4.054396806108321,
    4.054513925556535,
    4.054631045004748,
    4.0547481644529615,
    4.054865283901175,
    4.054982403349388,
    4.055099522797602,
    4.055216642245815,
    4.055333761694029,
    4.055450881142242,
    4.055568000590456,
    4.055685120038669,
    4.055802239486883,
    4.055919358935096,
    4.05603647838331,
    4.056153597831523,
    4.0562707172797365,
    4.05638783672795,
    4.0565049561761635,
    4.056622075624377,
    4.05673919507259,
    4.056856314520804,
    4.056973433969017,
    4.057090553417231,
    4.057207672865444,
    4.057324792313658,
    4.057441911761871,
    4.057559031210085,
    4.057676150658298,
    4.057793270106512,
    4.057910389554725,
    4.0580275090029385,
    4.058144628451152,
    4.0582617478993654,
    4.058378867347579,
    4.058495986795792,
    4.058613106244006,
    4.058730225692219,
    4.058847345140433,
    4.058964464588646,
    4.05908158403686,
    4.059198703485073,
    4.059315822933287,
    4.0594329423815,
    4.059550061829714,
    4.059667181277927,
    4.0597843007261405,
    4.059901420174354,
    4.060018539622567,
    4.060135659070781,
    4.060252778518994,
    4.060369897967208,
    4.060487017415421,
    4.060604136863635,
    4.060721256311848,
    4.060838375760062,
    4.060955495208275,
    4.061072614656489,
    4.061189734104702,
    4.0613068535529155,
    4.061423973001129,
    4.0615410924493425,
    4.061658211897556,
    4.061775331345769,
    4.061892450793983,
    4.062009570242196,
    4.06212668969041,
    4.062243809138623,
    4.062360928586837,
    4.06247804803505,
    4.062595167483264,
    4.062712286931477,
    4.062829406379691,
    4.062946525827904,
    4.0630636452761175,
    4.063180764724331,
    4.0632978841725444,
    4.063415003620758,
    4.063532123068971,
    4.063649242517185,
    4.063766361965398,
    4.063883481413612,
    4.064000600861825,
    4.064117720310039,
    4.064234839758252,
    4.064351959206466,
    4.064469078654679,
    4.064586198102893,
    4.064703317551106,
    4.0648204369993195,
    4.064937556447533,
    4.065054675895746
  ],
  "y": [
    309628,
    261303,
    241163,
    271546,
    297025,
    276946,
    256598,
    285970,
    329278,
    328297,
    302491,
    305150,
    324240,
    314676,
    294653,
    315970,
    360641,
    364051,
    331681,
    334311,
    382375,
    403104,
    364413,
    330667,
    357925,
    410421,
    426848,
    408606,
    395139,
    399441,
    411776,
    429265,
    444846,
    441529,
    427675,
    440077,
    483334,
    509731,
    493384,
    477682,
    505034,
    544686,
    547162,
    533760,
    563585,
    631881,
    673697,
    667022,
    663478,
    701973,
    752267,
    772625,
    776442,
    804092,
    858318,
    905921,
    929554,
    945063,
    971156,
    1007523,
    1045787,
    1083622,
    1120829,
    1152311,
    1173973,
    1192796,
    1224028,
    1275579,
    1338047,
    1393977,
    1435366,
    1467119,
    1495840,
    1524980,
    1561444,
    1613487,
    1675384,
    1726063,
    1753361,
    1771947,
    1808145,
    1873220,
    1957055,
    2038044,
    2094393,
    2122033,
    2150555,
    2220695,
    2330113,
    2424672,
    2468192,
    2497407,
    2571275,
    2684307,
    2779259,
    2840275,
    2916706,
    3041177,
    3177832,
    3281905,
    3367946,
    3480088,
    3625270,
    3778238,
    3932676,
    4104127,
    4291610,
    4476512,
    4658556,
    4859614,
    5090297,
    5340884,
    5606029,
    5890900,
    6190644,
    6491073,
    6795615,
    7127719,
    7499033,
    7894895,
    8297895,
    8708805,
    9139757,
    9602813,
    10108021,
    10657412,
    11240735,
    11852868,
    12511919,
    13241037,
    14038595,
    14891985,
    15816436,
    16849596,
    18009041,
    19290272,
    20710982,
    22320827,
    24159022,
    26239976,
    28592595,
    31273041,
    34320693,
    37743656,
    41572581,
    45884924,
    50739242,
    56121703,
    61996900,
    68374458,
    75273006,
    82644529,
    90392979,
    98454567,
    106802124,
    115375957,
    124071136,
    132800732,
    141514483,
    150134968,
    158523143,
    166534166,
    174070784,
    181055084,
    187382801,
    192944364,
    197674609,
    201547049,
    204523395,
    206535773,
    207522264,
    207461787,
    206365097,
    204239010,
    201071217,
    196859822,
    191660279,
    185595375,
    178813128,
    171442595,
    163596580,
    155401132,
    146998939,
    138518810,
    130045487,
    121612504,
    113222327,
    104891846,
    96690765,
    88722271,
    81062475,
    73748132,
    66834218,
    60416836,
    54561342,
    49244952,
    44418363,
    40090662,
    36290939,
    32969482,
    30013275,
    27357496,
    25009527,
    22956317,
    21124477,
    19456321,
    17957431,
    16635000,
    15447496,
    14353306,
    13357743,
    12471030,
    11664286,
    10907738,
    10211409,
    9586608,
    9006039,
    8445331,
    7926980,
    7475709,
    7062893,
    6646374,
    6239583,
    5888325,
    5595590,
    5318817,
    5032485,
    4751367,
    4498522,
    4284119,
    4106386,
    3944686,
    3766315,
    3567245,
    3385952,
    3249588,
    3133451,
    3005558,
    2878641,
    2777023,
    2684727,
    2575883,
    2467926,
    2390594,
    2327893,
    2248684,
    2170344,
    2127246,
    2096796,
    2029706,
    1941068,
    1892321,
    1884789,
    1853564,
    1777990,
    1716513,
    1707303,
    1709418,
    1677195,
    1627329,
    1592526,
    1565935,
    1534010,
    1514258,
    1515782,
    1505312,
    1457738,
    1404524,
    1387721,
    1395985,
    1388688,
    1358177,
    1327605,
    1307847,
    1294072,
    1287691,
    1288158,
    1279037,
    1252388,
    1230386,
    1233798,
    1246061,
    1238741,
    1217928,
    1214461,
    1236968,
    1261448,
    1263356,
    1242019,
    1218528,
    1219488,
    1252158,
    1290662,
    1304430,
    1301622,
    1316597,
    1352580,
    1377031,
    1383762,
    1410475,
    1471417,
    1523490,
    1535652,
    1544768,
    1595499,
    1665698,
    1707637,
    1731534,
    1785111,
    1869046,
    1936855,
    1973120,
    2012916,
    2080077,
    2157307,
    2225983,
    2292697,
    2366168,
    2442145,
    2519008,
    2601561,
    2689066,
    2783931,
    2902429,
    3046399,
    3180091,
    3276328,
    3371407,
    3522926,
    3720951,
    3900361,
    4044326,
    4211486,
    4441715,
    4695263,
    4919092,
    5125660,
    5364648,
    5649365,
    5952352,
    6254769,
    6564986,
    6897411,
    7257563,
    7643243,
    8048175,
    8472509,
    8932181,
    9441369,
    9987936,
    10554327,
    11162799,
    11861872,
    12660715,
    13521987,
    14437941,
    15466473,
    16661699,
    18014488,
    19505688,
    21181600,
    23122912,
    25361148,
    27881064,
    30694730,
    33858734,
    37417878,
    41380534,
    45753716,
    50552235,
    55756703,
    61305292,
    67150945,
    73292494,
    79720632,
    86359587,
    93096872,
    99857997,
    106618434,
    113342481,
    119938875,
    126287209,
    132290631,
    137888615,
    143027306,
    147635573,
    151635450,
    154966193,
    157589379,
    159475243,
    160593960,
    160918540,
    160425549,
    159093514,
    156914523,
    153912976,
    150144626,
    145678176,
    140594248,
    135003800,
    129040612,
    122816196,
    116396012,
    109834167,
    103210974,
    96610426,
    90073108,
    83600745,
    77204700,
    70928585,
    64832072,
    58982656,
    53460182,
    48333801,
    43624253,
    39314052,
    35400059,
    31904950,
    28825782,
    26105591,
    23679535,
    21523709,
    19632736,
    17972225,
    16487153,
    15146107,
    13946352,
    12881315,
    11931931,
    11086374,
    10338119,
    9662322,
    9022773,
    8409988,
    7847920,
    7352689,
    6905999,
    6481733,
    6079526,
    5714825,
    5387805,
    5079809,
    4777353,
    4488213,
    4232630,
    4020163,
    3833015,
    3637351,
    3420822,
    3211826,
    3043905,
    2907319,
    2763388,
    2606921,
    2478076,
    2397983,
    2328716,
    2225381,
    2100140,
    1998805,
    1931989,
    1868517,
    1788966,
    1707375,
    1638892,
    1578950,
    1520419,
    1465383,
    1414342,
    1364300,
    1318487,
    1278164,
    1229870,
    1165998,
    1111017,
    1094068,
    1098763,
    1078637,
    1024619,
    975261,
    952286,
    931407,
    893308,
    857716,
    841741,
    826875,
    798098,
    775757,
    772621,
    757956,
    708203,
    658757,
    653213,
    668593,
    650116,
    601945,
    578031,
    588086,
    580850,
    532114,
    486056,
    482778,
    496425,
    480511,
    437532,
    406371,
    404514,
    416312,
    423533,
    419148,
    400247,
    370481,
    344314,
    332401,
    328072,
    320489,
    313732,
    319848
  ],
  "id": "gpZJPOQ4",
  "from": 4.004522423699725,
  "to": 4.06513338174316,
  "absolute": 1693176.4207008644,
  "signal": [
    {
      "multiplicity": "",
      "kind": "signal",
      "delta": 4.024443507227576,
      "j": []
    }
  ],
  "integral": 100
}
```

**Comment by djeanner** [comment]
Difficult problem: The multiplet is not symmetrical. Determining the center of a multiplet is the first step of the analysis and if fails in this case, or more precisely determines one of the line of the doublet to be the multiplet satisfying the symetry criterion.

**Comment by djeanner** [comment]
Possible solution: reject, in the search of the center of a multiplet, any position that would not cover 90% of the integration of the area. This shoul allow doublet with pretty severe second-order effects to be taken as a whole.

**Comment by djeanner** [comment]
Problem solved with commit 46ed8ad
